modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -101,30 +156,6 @@ impl SbomContext {
 
         creator.create(&db).await?;
 
-        // prepare relationships
```

Comment:
Why do we remove relationships?

Reply:
Ah I see it was moved to the top.

Reply:
Yeah, it's just moved above packages, so we can use ids to act on the package processing